tests/integration/_experimental/object_detection/test_evaluator_multiclass_fixed.py

```diff
@@ -882,44 +836,50 @@ def assert_test_case_plots_equals_expected(
 
 @pytest.mark.metrics
 def test__object_detection__multiclass_evaluator__fixed() -> None:
-    from kolena._experimental.object_detection import ObjectDetectionEvaluator
-
     config = ThresholdConfiguration(
-        threshold_strategy=ThresholdStrategy.FIXED_05,
+        threshold_strategy=0.5,
         iou_threshold=0.5,
         min_confidence_score=0,
-        with_class_level_metrics=True,
     )
-    eval = ObjectDetectionEvaluator(configurations=[config])
+    eval_fixed = ObjectDetectionEvaluator(configurations=[config])
 
-    test_sample_metrics = eval.compute_test_sample_metrics(
+    test_sample_metrics = eval_fixed.compute_test_sample_metrics(
         test_case=TEST_CASE,
         inferences=TEST_DATA,
         configuration=config,
     )
 
-    assert len(eval.evaluator.threshold_cache) == 0  # empty because not f1 optimal config
-    assert len(eval.evaluator.matchings_by_test_case) != 0
-    assert len(eval.evaluator.matchings_by_test_case[config.display_name()]) != 0
-    assert len(eval.evaluator.matchings_by_test_case[config.display_name()][TEST_CASE.name]) == len(TEST_DATA)
+    print(config)
```

Comment:
Are these intentional?

Reply:
The prints will be removed (was debugging CI behaviour)